include/rang.hpp

```diff
@@ -88,7 +89,13 @@ enum class bgB {
 	gray    = 107
 };
 
-enum class control { autoColor = 0, forceColor = 1 };
+enum class control {
+    autoColor = 0,
+    forceColor = 1,
+    bufferInput = 2,
```

Comment:
My apologies that slipped in. That is part of my input work. I am developing it in a separate pr. I will revert these enums tomorrow. That's not what they ended up being anyway.